client/src/app/app.component.scss

```diff
@@ -0,0 +1,37 @@
+// Standard Styles
+body {
+    /* Margin bottom by footer height */
+    margin-bottom: 60px;
+  }
```

Comment:
Looks like you've got some mismatched indentation in this file. VSCode will take care of this for you by auto-formatting on save if you change this setting in your [VSCode Preferences](https://code.visualstudio.com/docs/getstarted/settings):
```
 // Format a file on save. A formatter must be available, the file must not be auto-saved, and editor must not be shutting down.
  "editor.formatOnSave": false,
```